.github/workflows/ecs_prod.yml

```diff
@@ -0,0 +1,77 @@
+name: api-polygon-tokens-ecs
+on:
+  push:
+    branches: ['master']
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: api-polygon-tokens-prod-ecs
+    permissions:
+      id-token: write
+      contents: write
+    environment: prod
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: eu-west-1
+      ECR_REPOSITORY: api-polygon-tokens-ecr
+      ECS_SERVICE: api-polygon-tokens-ecs-service
+      ECS_CLUSTER: frontend-prod-ecs-cluster
+      ECS_TASK_DEFINITION: prod-taskdef.json
+      CONTAINER_NAME: api-polygon-tokens
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+
+      # Runs a set of commands using the runners shell
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build tokenlists
+        run: npm run build
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
Change the version to V2 as v1-node16 will give you error

Reply:
updated to v2